.github/workflows/update_dev_version.yml

```diff
@@ -0,0 +1,38 @@
+name: Update dev version automatically
+on:
+  pull_request:
+
+jobs:
+  update-dev-version:
+    name: Update version for development branches
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout PennyLane-Lightning-GPU master
+        uses: actions/checkout@v2
+        with:
+          ref: main
+          path: main
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Update package version tag
+        uses: actions/checkout@v3
```

Comment:
It's good to also update this part in PennyLane-Lightning.

Reply:
Yep, agreed